cypress.config.ts

```diff
@@ -1,7 +1,22 @@
 import { defineConfig } from 'cypress';
+import dotenv from 'dotenv';
+
+dotenv.config({
+  path: `.env`,
+});
+dotenv.config({
+  path: `.env.local`,
+  override: true,
+});
 
 export default defineConfig({
+  env: {
+    codeCoverage: {
+      exclude: ['cypress/**/*.*', 'node_modules/**'],
```

Comment:
If not, then you should do it before merge.

Reply:
Yes , it is up to date with develop

Reply:
That cypress config lacks the shadcn folder ignore

![image](https://github.com/AlanMamphs/graphiql-app/assets/65208028/313da24f-bcf7-4288-8f44-863d4d9e7924)

Reply:
I removed it because we get the required 90% coverage of our UI components. We may want to delete unused components at the end of the project to improve the coverage.

Reply:
<img width="850" alt="image" src="https://github.com/AlanMamphs/graphiql-app/assets/14315501/0a28a02d-3da3-4e19-8150-ab91560a2e5c">